src/shared/services/dom-util.service.ts

```diff
@@ -32,6 +32,35 @@ export class DomUtilService {
       if (firstInput) {
         firstInput.focus();
         firstInput.select();
+      } else {
+        firstInput = el.querySelector('div.btn-group') as HTMLInputElement;
+        if (firstInput) {
+          // .open is used for opening the dropdownmenu
+          firstInput.classList.add('open');
+          firstInput = el.querySelector('div.btn-group input') as HTMLInputElement;
+          firstInput.focus();
```

Comment:
@jmartinm i don't think this is the same logic as before. If first input is undefined everywhere it will attempt to focus and select the undefined element.